src/components/github-stars.jsx

```diff
@@ -12,12 +11,11 @@ if (typeof window !== 'undefined') window.githubStars = githubStars;
 
 export default class GithubStars extends Component {
 	state = {
-		stars: localStorageGet('_stars') || fallbackData.preactStargazers
```

Comment:
Getting/setting from/to localstorage here seems unnecessary? I don't see any obvious benefit to doing so.